src/VariableStatementAnalysis.ts

```diff
@@ -1,3 +1,4 @@
 export interface VariableStatementAnalysis {
```

Comment:
It can be helpful to get the line and starting character of the expression and variable identifier.

```typescript
node.pos
node.end

ts.getLineAndCharacterOfPosition
```

Reply:
Added